src/sections/FAQ/index.tsx

```diff
@@ -0,0 +1,17 @@
+import Typography from '@/components/Typography';
+import { QUESTIONS } from './questions';
+import styles from './style.module.scss';
+import FAQAccordion from '@/components/FAQAccordion';
+
+const FAQ: React.FC = () => {
+  return (
+    <section className={styles.container}>
+      <Typography variant="display/heavy/small" className={styles.title}>
+        Frequently Asked Questions
+      </Typography>
+      <FAQAccordion data={QUESTIONS} />
```

Comment:
I think maybe it would be better if each accordion was one component; that way the component is more reusable. You could handle enforcing only one accordion open in this component, or let each accordion be independently openable

Reply:
We could do that in this component, but I'd prefer doing it in `FAQAccordion` so we can have all the MUI work constrained to just one file

Reply:
> I think maybe it would be better if each accordion was one component

That's probably the best (and maybe something we should do if we were writing this in a design system, but we're not going to be using accordion anywhere else in this site and I'd rather just rip through everything